app/assets/stylesheets/main.scss

```diff
@@ -1,3 +1,6 @@
+//sass tools
+@import "variables";
```

Comment:
This is not needed. We're already have this line `@import "helpers/variables"`. No need for another variables file.